src/markdown-pages/docs/cli-command-reference/simbank-cli.md

```diff
@@ -0,0 +1,116 @@
+---
+path: "/docs/cli-command-reference/simbank-cli"
+title: "Exploring Galasa SimBank using the CLI"
+---
+Distributed with Galasa, SimBank is a component that simulates a mainframe application. It sits above another component called SimPlatform, which exists to provide options for future growth. As delivered, SimBank implements a sample banking application against which you can configure and run a set of provided tests in preparation for running your own tests against an *actual* mainframe application. You can also practice writing some new tests to run against the SimBank banking application.
```

Comment:
"It sits above another component called SimPlatform, which exists to provide options for future growth." seems a bit vague.